### PR TITLE
feat: Add some code owner

### PR DESCRIPTION
**Pull request by yoyo837**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 🔗 Related issue link

ref: #19410、#19413

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Add some code owner       |
| 🇨🇳 Chinese |     添加部分 code owner  试用     |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,9 @@
 # A CODEOWNERS file uses a pattern that follows the same rules used in gitignore files.
 # The pattern is followed by one or more GitHub usernames or team names using the standard @username or @org/team-name format.
 # You can also refer to a user by an email address that has been added to their GitHub account, for example user@example.com.
+
+/*                                @afc163
```
Comment (afc163) [inline CODEOWNERS]: /*                                @zombieJ

File: CODEOWNERS [hunk 2]
```diff
@@ -2,3 +2,9 @@
 # A CODEOWNERS file uses a pattern that follows the same rules used in gitignore files.
 # The pattern is followed by one or more GitHub usernames or team names using the standard @username or @org/team-name format.
 # You can also refer to a user by an email address that has been added to their GitHub account, for example user@example.com.
+
+/*                                @zombieJ
```
Comment (yoyo837) [inline CODEOWNERS]: 要不干脆默认不指定owner ?

Comment (yoyo837) [reply CODEOWNERS]: 只指定组件的 onwer 其他不指定？

Comment (afc163) [reply CODEOWNERS]: 这样比较好。

**Pull request merged**